### PR TITLE
Issue #2857357 by vasike, bojanz, Berdir: Replace entity.query usage

### PR DESCRIPTION
**Pull request by vasike**

### CODE REVIEW

File: modules/order/src/Form/OrderTypeDeleteForm.php [hunk 1]
```diff
@@ -3,46 +3,18 @@
...
-      $container->get('entity.query')
-    );
-  }
-
-  /**
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    $order_count = $this->queryFactory->get('commerce_order')
+    $order_count = $order_item_count = $this->entityTypeManager->getStorage('commerce_order')->getQuery()
```
Comment (bojanz) [inline modules/order/src/Form/OrderTypeDeleteForm.php]: This line is messed up :)

File: modules/checkout/commerce_checkout.module [hunk 2]
```diff
@@ -26,6 +26,9 @@ function commerce_checkout_theme() {
     'commerce_checkout_form__with_summary' => [
       'base hook' => 'commerce_checkout_form',
     ],
+    'commerce_checkout_registration' => [
```
Comment (bojanz) [inline modules/checkout/commerce_checkout.module]: Why do we need this?

File: modules/order/src/Form/OrderTypeDeleteForm.php (hunk 1 above)
Comment (vasike) [reply modules/order/src/Form/OrderTypeDeleteForm.php]: copied too much ;)

File: modules/checkout/commerce_checkout.module (hunk 2 above)
Comment (vasike) [reply modules/checkout/commerce_checkout.module]: no need. from other issue, sorry

**Pull request merged**